src/main/java/com/epam/ta/reportportal/core/launch/impl/DeleteLaunchHandler.java

```diff
@@ -109,10 +109,10 @@ public OperationCompletionRS deleteLaunches(String[] ids, String projectName, St
 		final User user = userRepository.findOne(userName);
 		final Project project = projectRepository.findOne(projectName);
 		launches.forEach(launch -> validate(launch, user, project));
-		launchRepository.delete(toDelete);
 		launches.forEach(launch -> logIndexer.cleanIndex(projectName,
```

Comment:
what if logindex returns error? wouldn't be better to wrap it up with some try/catch?

Reply:
log index is wrapped with try/catch in the place where it could be error

https://github.com/reportportal/service-api/blob/25e094248b40b6a3a4a68451344b4d6a588633b3/src/main/java/com/epam/ta/reportportal/core/analyzer/client/AnalyzerServiceClient.java#L110